code/modules/food_and_drinks/recipes/tablecraft/recipes_meat.dm

```diff
@@ -81,6 +81,37 @@
 	result = /obj/item/reagent_containers/food/snacks/sashimi
 	subcategory = CAT_MEAT
 
+/datum/crafting_recipe/food/sushie_basic
+	name = "Funa Hosomaki"
+	reqs = list(
+		/datum/reagent/consumable/soysauce = 3,
+		/obj/item/reagent_containers/food/snacks/salad/ricebowl = 1,
+		/obj/item/reagent_containers/food/snacks/carpmeat = 2,
+		/obj/item/reagent_containers/food/snacks/grown/kudzupod = 3,
```

Comment:
So these'll be used about once a year since kudzu spawns are terribly rare and no botanists ever play with it because they know they'll get absolutely destroyed for self-antagging.

Reply:
Kuz is the flat out easiest weed to use, and its get-able 3 mins of round start